packages/instrumentation-amqplib/test/config.ts

```diff
@@ -2,4 +2,4 @@
 // 1. 'test:docker:run' script in package.json
 // 2. services in github actions workflow
 export const TEST_RABBITMQ_HOST = 'localhost';
-export const TEST_RABBITMQ_PORT = 22221;
+export const TEST_RABBITMQ_PORT = 5672; //22221;
```

Comment:
I use non-default port for testing so tests will not excitedly publish and consume to a local broker which is used for other things. 
This is correlated with [test workflow in CI](https://github.com/aspecto-io/opentelemetry-ext-js/blob/master/.github/workflows/test.yml#L36) and [local setup script](https://github.com/aspecto-io/opentelemetry-ext-js/blob/master/packages/instrumentation-amqplib/package.json#L35)
I think this is why tests are failing now.

Reply:
Ye, forgot to revert as I use default :) Thanks